application/models/Verification_Model.php

```diff
@@ -108,6 +108,44 @@ public function hasUserPaidEvent($emailAddress, $sheetName)
     }
 
 
+
+    /**
+     * Get user's full name, UPI and UID on the membership spreadsheet.
+     *
+     * @param string $emailAddress The email of the user.
+     *
+     * @return array [full_name, UPI, UID]
+     */
+    public function getUserInfo($emailAddress)
```

Comment:
Let's rename this to `getMemberInfo()`, since we're referring to each of ASPA's current people as members (for example, the sheet is called Membership Spread).